src/js/components/molecules/in-page-notification/in-page-notification.ts

```diff
@@ -2,16 +2,41 @@
 import { Component, Prop, Vue } from 'vue-property-decorator';
 import WithRender from './in-page-notification.vue.html';
 import { Notification as NotificationInterface } from '@ultimaker/ultimaker.com-model-definitions/dist/molecules/notification/Notification';
+import Events from 'constants/events';
 
 @WithRender
 @Component({
     name: 'InPageNotification',
 })
 export class InPageNotification extends Vue implements NotificationInterface {
-    @Prop({ type: String, required: true }) message!: NotificationInterface['message'];
+    @Prop({ type: String, required: true }) public message!: NotificationInterface['message'];
+    @Prop({ type: Object, required: false }) public clickEvent?: NotificationInterface['clickEvent'];
     public display: boolean = true;
```

Comment:
🐜 new line between @Prop and class variable

Reply:
i agree